spec/requests/shared_examples/api/v4/shared_spec_for_sync_requests.rb

```diff
@@ -0,0 +1,161 @@
+require 'rails_helper'
```

Comment:
Are these the legendary shared sync specs?

Reply:
_some_ of them

Reply:
My god